src/Cli/ConfigGenerator.cs

```diff
@@ -923,6 +943,149 @@ private static bool TryUpdateConfiguredCacheValues(
             }
         }
 
+        /// <summary>
+        /// Attempts to update the Config parameters in the Host runtime settings based on the provided value.
+        /// Validates that any user-provided parameter value is valid and then returns true if the updated Host options
+        /// needs to be overwritten on the existing config parameters
+        /// </summary>
+        /// <param name="options">options.</param>
+        /// <param name="updatedHostOptions">updatedHostOptions.</param>
+        /// <returns>True if the value needs to be udpated in the runtime config, else false</returns>
+        private static bool TryUpdateConfiguredHostValues(
+            ConfigureOptions options,
+            ref HostOptions? updatedHostOptions)
+        {
+            object? updatedValue;
+            try
+            {
+                // Runtime.Host.Mode
+                updatedValue = options?.RuntimeHostMode;
```

Comment:
please use more descriptive variable names, you can create different variable, `updatedValue` is being used her for different values, it gets a little confusing at times.

Reply:
The idea was to not create many variables and hence object is the general data type for updatedValue and that's being used for all primitive variables